src/content/guides/tree-shaking.mdx

```diff
@@ -361,7 +361,7 @@ What we've learned is that in order to take advantage of _tree shaking_, you mus
 - Use ES2015 module syntax (i.e. `import` and `export`).
 - Ensure no compilers transform your ES2015 module syntax into CommonJS modules (this is the default behavior of the popular Babel preset @babel/preset-env - see the [documentation](https://babeljs.io/docs/en/babel-preset-env#modules) for more details).
 - Add a `"sideEffects"` property to your project's `package.json` file.
-- Use the [`production`](/configuration/mode/#mode-production) `mode` configuration option to enable [various optimizations](/configuration/mode/#usage) including minification and tree shaking.
```

Comment:
Let's keep the original sentence, but just add a note that side effects optimization is enabled in development mode using the `flag` value